interpreters/duktape/Makefile

```diff
@@ -48,23 +48,31 @@ CFLAGS += -I$(DUKTAPE_UNPACK)/extras/print-alert
 CFLAGS += -DDUK_CMDLINE_CONSOLE_SUPPORT
 CFLAGS += -DDUK_CMDLINE_PRINTALERT_SUPPORT
 
+APPINC = $(CURDIR)/$(DUKTAPE_UNPACK)
+
 $(DUKTAPE_TARBALL):
-	@echo "Downloading $(DUKTAPE_TARBALL)"
-	@wget $(DUKTAPE_URL)
+	$(Q) echo "Downloading $(DUKTAPE_TARBALL)"
+	$(Q) wget $(DUKTAPE_URL)
 
 $(DUKTAPE_UNPACK): $(DUKTAPE_TARBALL)
-	@echo "Unpacking $(DUKTAPE_TARBALL) to $(DUKTAPE_UNPACK)"
-	@tar xvfJ $(DUKTAPE_TARBALL)
-	@echo "Patching $(DUKTAPE_UNPACK)"
-	@patch -p0 < duk_cmdline.patch
-	@touch $(DUKTAPE_UNPACK)/.patch
+	$(Q) echo "Unpacking $(DUKTAPE_TARBALL) to $(DUKTAPE_UNPACK)"
+	$(Q) tar xvfJ $(DUKTAPE_TARBALL)
+	$(Q) ln -sf $(DUKTAPE_UNPACK) duktape
```

Comment:
let's define DUKTAPE_UNPACK=duktape and mv duktape-2.5.0 to duktape directly instead link